src/App-au.vue

```diff
@@ -90,10 +53,12 @@ export default defineComponent({
       }
     },
   },
-  created() {
-    fetchData('widget', {
+  async created() {
+    const responseData = (await (fetchData('widget', {
       method: 'POST',
-    })
+    }) as unknown)) as ResponseProps
+    this.terms = responseData.terms
+    this.tabs = getTabsData(responseData.products)
```

Comment:
This is repeated a few times, could you please create a separate fetchWidgetData function that you reuse instead? 
Also, see if you can make fetchData generic so that you can pass in a type assertion instead of converting to unknown then ResponseProps.

Basic example that can be expanded with guards, error handling and more: 

Please change the passed in typings to an appropriate model
```
async created() {
    const response = await fetchData<{ terms: string; products: string[] }>(
      'widget',
      {
        method: 'POST',
      }
    )

    this.terms = response.terms
    this.tabs = getTabsData(response.products)
  },
```

```
export async function fetchData<T>(
  endpoint: string,
  config?: RequestInit
): Promise<T> {
  const request = await fetch(`${apiUrl}/${endpoint}`, config)
  const response = request.json()
  return response
}
```